admin_guide/manage_nodes.adoc

```diff
@@ -146,76 +156,118 @@ Pods are not scheduled for the node until the node passes the health checks that
-****
+----
+$ oadm manage-node --selector=<node_selector> \
+--list-pods [--pod-selector=<pod_selector>] [-o json|yaml]
+----
 
 == Marking Nodes as Unschedulable or Schedulable
-Use the following command to mark the nodes as unschedulable:
+[[marking-nodes-as-unschedulable-or-schedulable]]
+By default, healthy nodes with a `Ready` link:#node-conditions[status] are
+marked as schedulable, meaning that new pods are allowed for placement on the
+node. Manually marking a node as unschedulable blocks any new pods from being
+scheduled on the node. Existing pods on the node are not affected.
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe=false`
-****
+To mark a node or nodes as unschedulable:
 
-Marking node as unschedulable will block any new pods to be scheduled on the
-node. Existing pods on the node will not get affected.
+----
+$ oadm manage-node <node1> <node2> --schedulabe=false
+----
 
-Use the following command to mark the nodes as schedulable:
+For example:
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe` or
+====
+[options="nowrap"]
+----
+$ oadm manage-node node1.example.com --schedulable=false
+NAME                 LABELS                                        STATUS
+node1.example.com    kubernetes.io/hostname=node1.example.com      Ready,SchedulingDisabled
+----
+====
+
+To mark a currently unschedulable node or nodes as schedulable:
 
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe=true`
-****
+----
+$ oadm manage-node <node1> <node2> --schedulabe
+----
 
-Marking node as schedulable will allow any new pods to be scheduled on the
-node.
+Or:
+
+----
+$ oadm manage-node <node1> <node2> --schedulabe=true
+----
 
-Instead of specifying the nodes _<node1>_ _<node2>_, you can also use
---selector=_<node_selector>_ to mark selected nodes as schedulabe or
-unschedulable.
+Alternatively, instead of specifying specific node names (e.g., `_<node1>_
+_<node2>_`), you can use the `--selector=_<node_selector>_` option to mark
+selected nodes as schedulabe or unschedulable.
 
 == Evacuating Pods on Nodes
-Use the following command to evacuate all or selected pods on one or more nodes:
+Nodes must first be link:#marking-nodes-as-unschedulable-or-schedulable[marked
+unschedulable] to perform pod evacuation. Only pods backed by a
+link:../architecture/core_objects/kubernetes_model.html#replication-controller[replication
+controller] are evacuated. Bare pods are unaffected by default.
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --evacuate [--pod-selector=_<pod_selector>_]`
-****
+To list pods that will be migrated without actually performing the evacuation,
+use the `--dry-run` option:
 
-Nodes must be marked unschedulable to perform pod evacuation. Only pods backed
-by replication controller will be evacuated. Bare pods will not be touched.
-You can force deletion of bare pods by using --force option:
+----
+$ oadm manage-node <node1> <node2> \
+--evacuate --dry-run [--pod-selector=<pod_selector>]
```

Comment:
I know the writing guide suggests flush-left continuation lines (after backslash), so this is just a style suggestion: add a few spaces (like 4).